README.md

```diff
@@ -204,16 +201,10 @@ for (Bucket bucket : storage.list().iterateAll()) {
 
 #### Complete source code
 
-In
-[CreateAndListBucketsAndBlobs.java](https://github.com/googleapis/google-cloud-java/tree/main/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/CreateAndListBucketsAndBlobs.java)
-we put together examples creating and listing buckets and blobs into one program. The program assumes that you are
-running on Compute Engine or from your own desktop. To run the example on App Engine, simply move
-the code from the main method to your application's servlet class and change the print statements to
-display on your webpage.
+See [ListObjects.java](https://github.com/googleapis/java-storage/blob/main/samples/snippets/src/main/java/com/example/storage/object/ListObjects.java) for a complete example.
```

Comment:
Was this the regenerated changes? Not sure if they will be clobbered once merged again.

Reply:
Yes, these are the result of owlbot updates after i edited the yaml file, so these changes are safe from clobbering